torch/testing/_internal/mps_utils.py

```diff
@@ -0,0 +1,1012 @@
+            "igamma",
+            "igammac",
+            "index_copy",
+            "index_reduce",
+            "kthvalue",
+            "lcm",
+            "linalg.cond",
+            "linalg.eigh",
+            "linalg.eigvalsh",
+            "linalg.householder_product",
+            "linalg.ldl_factor",
+            "linalg.ldl_factor_ex",
+            "linalg.ldl_solve",
+            "linalg.lstsq",
+            "linalg.lstsqgrad_oriented",
+            "linalg.lu",
+            "linalg.lu_solve",
+            "linalg.qr",
+            "linalg.svdvals",
+            "linalg.vecdot",
+            "logcumsumexp",
+            "lu_solve",
+            "masked.median",
+            "matrix_exp",
+            "mode",
+            "native_dropout_backward",
+            "nn.functional.fractional_max_pool2d",
+            "nn.functional.fractional_max_pool3d",
+            "nn.functional.adaptive_avg_pool3d",
+            "nn.functional.adaptive_max_pool3d",
+            "nn.functional.max_unpool1dgrad",
+            "nn.functional.max_unpool2dgrad",
+            "nn.functional.max_unpool3dgrad",
+            "nn.functional.avg_pool3d",
+            "nn.functional.ctc_loss",
+            "nn.functional.embedding_bag",
+            "nn.functional.hardshrink",
+            "nn.functional.max_pool3d",
+            "nn.functional.max_unpool1d",
+            "nn.functional.max_unpool2d",
+            "nn.functional.max_unpool3d",
+            "nn.functional.multi_margin_loss",
+            "nn.functional.multilabel_margin_loss",
+            "nn.functional.pdist",
+            "nn.functional.rrelu",
+            "nn.functional.norm",
+            "ormqr",
+            "pca_lowrank",
+            "qr",
+            "rsub",
+            "segment_reduce",
+            "_segment.reduce",
+            "segment.reduce",
+            "segment_reduce_offsets",
+            "_segment_reduce",
+            "_segment_reducelengths",
+            "_segment_reduceoffsets",
+            "sparse.mm",
+            "sparse.mmreduce",
+            "special.airy_ai",
+            "special.erfcx",
+            "special.hermite_polynomial_he",
+            "special.laguerre_polynomial_l",
+            "special.log_ndtr",
+            "special.ndtri",
+            "svd_lowrank",
+            "symeig",
+            "take",
+            "to",
+            "to_sparse",
+            "unique",
+            "vdot",
+            "segment_reduce_",
+            "geometric",
+            "geometric_",
+            "log_normal_",
+            "log_normal",
+            "cdouble",
+            "double",
+            "float_power",
+            "full_like",
+            "nonzero_static",
+            "nn.functional.adaptive_avg_pool1d",
+            "nn.functional.adaptive_avg_pool2d",
+        ]
+    },
+    **{
+        "linalg.matrix_norm": MPSSkipInfo(UNIMPLEMENTED, dtypes=[torch.float32]),
+        "linalg.norm": MPSSkipInfo(UNIMPLEMENTED, dtypes=[torch.float32]),
```

Comment:
Can you help me understand what it means? That `linalg.norm` is unimplemented for float32, but works for float16?